sentry/src/main/java/io/sentry/SentryClient.java

```diff
@@ -172,6 +182,28 @@ private boolean shouldApplyScopeData(
     return sentryId;
   }
 
+  private boolean shouldSendSessionUpdateForDroppedEvent(
```

Comment:
@bruno-garcia I assume we don't want to send if the User Agent changed here, right? Not sure if a change of the User Agent can even happen for user attended sessions.

Reply:
I don't think user agent is used at all. It was part of the first design but didn't make it to the product.
These values such as environment, release, user-agent shouldn't change throughout a session so lets not consider those